scripts/couchdb/utils.py

```diff
@@ -34,7 +35,7 @@ def get_membership(node_details):
 
 
 def confirm(msg):
-    return raw_input(msg + "\n(y/n)") == 'y'
+    return input(msg + "\n(y/n)") == 'y'
```

Comment:
@dannyroberts should scripts/couchdb get removed from HQ since you created that other repo?

Reply:
Oh. Yeah, totally. Man I've been forgetting things like it's my job. I'll go clean that up

Reply:
https://github.com/dimagi/commcare-hq/pull/18498